tests/test_precision.py

```diff
@@ -161,6 +187,14 @@ def test_snippets():
     npt.assert_almost_equal(ref_areas, cmp_areas, decimal=config.STUMPY_TEST_PRECISION)
     npt.assert_almost_equal(ref_regimes, cmp_regimes)
 
+    if not numba.config.DISABLE_JIT:  # pragma: no cover
+        # Revert fastmath flag back to their default values
+        config._reset("STUMPY_FASTMATH_FLAGS")
+        core._calculate_squared_distance.targetoptions["fastmath"] = (
```

Comment:
@seanlaw 
Using a config variable to set the fastmath flags for this function feels a bit weird here. This goes back to the concern you previously raised that "how should one know what the default value is?" Alternatively, we can store the default fasthmath flags for this function before changing it and using it when reverting the change. Or, as discussed previously, we can just say the current approach is good enough for this rare situation.

Reply:
Yes, see my comment above.